### PR TITLE
Fizzbuzz with if, for done

### LINKED ISSUE #1: Do fizzbuzz with if, for
## Task List

- [x] Do fizz
- [x] Do buzz
- [x] Do fizzbuzz

### PR DESCRIPTION
**Pull request by cona-tus**
close #1

### CODE REVIEW

File: fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+for i in range(1,16+1):
+	if i%15==0:
```
Comment (Doyeon-97) [inline fizzbuzz.py:2]: Could you do this with 1 if?

File: fizzbuzz.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+for i in range(1,16+1):
```
Comment (Doyeon-97) [inline fizzbuzz.py:1]: Set num range 16 to 31

**Review by Doyeon-97 — CHANGES_REQUESTED** [review]

File: fizzbuzz.py (hunk 1 above)
Comment (cona-tus) [reply fizzbuzz.py:2]: No problem 🥲

File: fizzbuzz.py (hunk 2 above)
Comment (cona-tus) [reply fizzbuzz.py]: Done!

**Review by Doyeon-97 — APPROVED** [review]
Well done

**Pull request merged**